MapboxDirections/Match/MBTracepoint.swift

```diff
@@ -0,0 +1,41 @@
+import Foundation
+
+@objc(MBTracepoint)
+public class Tracepoint: Waypoint {
+    
+    /**
+     Number of probable alternative matchings for this trace point. A value of zero indicates that this point was matched unambiguously.
+     */
+    @objc open var alternateCount: Int
+    
+    /**
+     Index of the waypoint inside the matched route.
+     */
+    open var waypointIndex: Int?
```

Comment:
Sadly, this can be nil. @1ec5 how can we make this bridge to objc?

Reply:
How will developers tend to encounter this property? When a developer finds themselves needing to associate a tracepoint with a route, do they already have a route on hand? If so, consider making the route hold a reference to a tracepoint somehow.

(`NSNotFound` represents a nonexistent index.)